sdk/core/azure-core-amqp/vcpkg/Config.cmake.in

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(azure-core-cpp "1.9.0-beta.1")
```

Comment:
Eventhubs `Config.cmake.in` depends on 1.8.0, should we be consistent and depend on that version here too, given eventhubs will depend on amqp which depends on core? I suspect you can't because of the uuid changes, correct?

Discussion on that file
https://github.com/Azure/azure-sdk-for-cpp/pull/4596#discussion_r1184477429

Reply:
@gearama

Reply:
#4609